### ISSUE TITLE
Is this extension still needed?

### ISSUE THREAD

**Issue by sannidhyaroy** [Dragicafit/Crunchyroll-With-Better-Seasons#21]
I've been using this extension for a few weeks and today I noticed that if I try playing a series on Crunchyroll, the video player will continue to load infinitely. If I turn off the extension, the player loads. I tested this for multiple series on two browsers (Arc and Chrome) and there definitely seems to be an issue. Probably Crunchyroll updated their site, which renderer this extension broken.

Did they implement this feature natively in their site? I'm confused what's going on!

**Comment by Lolen10** [comment]
Yes. They implemented it natively. You probably won't need this extension anymore.

**Comment by Dragicafit** [comment]
I disabled the extension for now, I will update the extension depending on what they support.

I will maintain the extension with at least the ability to select the subtitles from the Japanese audio in other languages.

**Comment by sannidhyaroy** [comment]
> I disabled the extension for now, I will update the extension depending on what they support.

Took Crunchyroll a lot of time to get this natively! Thanks for making this extension, it was quite helpful!

**Comment by LavaCubeL** [comment]
Wait a second Crunchyroll only adds subtitles to English dubs. Even though they are starting to add subtitles it's still a work in progress I guess I have to wait for support or develop my own chrome extension/workaround for this issue.

**Comment by Dragicafit** [comment]
So I released the extension with the ability to choose the subtitles in all dubs and the ability to know which seasons has which language directly in the selection of the seasons
